src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
HTML Content: The HTML file lacks semantic tags. The `div` tag is used excessively. Consider using more semantic tags like `section`, `article`, etc., to improve your page's SEO and help screen readers. `div` and `span` do not have any meaning.